src/pugixml.cpp

```diff
@@ -6077,6 +6094,22 @@ namespace pugi
 		return true;
 	}
 
+	PUGI__FN bool xml_node::remove_children()
```

Comment:
This function will crash when xml_node is a null handle, this needs a null check for _root and a test that xml_node().remove_children() works.